zipkin/src/main/java/zipkin2/storage/InMemoryStorage.java

```diff
@@ -77,15 +77,12 @@ public static Builder newBuilder() {
     int maxSpanCount = 500000;
     List<String> autocompleteKeys = Collections.emptyList();
 
-    /** {@inheritDoc} */
-    @Override
-    public Builder strictTraceId(boolean strictTraceId) {
+    @Override public Builder strictTraceId(boolean strictTraceId) {
```

Comment:
This strategy seems good. Just wondering do you think this is a good time to move stuff out of core? For example, I guess storage, since it's for use by server and not client, doesn't need to be Java 6?

Reply:
IIRC we might have some senders that delegate to storage classes (thinking zipkin-aws). I think it could be a net win to duplicate for this rare case and mark storage as server-only.

Reply:
yeah we currently only support java 1.6 on the core library. the storage classes are already 1.8 iirc. that zipkin-aws xray sender internally uses storage is something that we aren't optimizing for in a bytecode level..

typically, senders raise to the level of the library used, sometimes 1.7 (okhttp) or 1.8 (kafka)

Reply:
oh I see what you are saying.. to move these classes out? I think it is a non-transparent change to do that where as the rest of this is transparent to end users. I think we could consider doing that separately, but there would be some transition work involved and so not entirely keen to take that on right now.

Reply:
https://github.com/openzipkin/zipkin/issues/3217